tap_facebook/__init__.py

```diff
@@ -196,10 +208,12 @@ def do_request():
             include_deleted = CONFIG.get('include_deleted', 'false')
             if include_deleted.lower() == 'true':
                 params.update({'filtering': get_delivery_info_filter('ad')})
-            return self.account.get_ads(fields=self.fields(), params=params) # pylint: disable=no-member
+            if self.current_bookmark:
+                params['filtering'].append({'field': 'ad.' + UPDATED_TIME_KEY, 'operator': 'GREATER_THAN', 'value': self.current_bookmark.int_timestamp})
```

Comment:
`params['filtering']` might not exist at this point if `include_deleted` is != "true".

Also, If this works out, the checking in IncrementalStream._iterate wouldn't be necessary, right?

Reply:
Good catch